### PR TITLE
OPENEUROPA-646: Add demo landing page node content type

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

File: behat.yml.dist [hunk 1]
```diff
@@ -18,6 +18,8 @@ default:
               language switcher overlay: ".ecl-dialog__body"
               language switcher link: ".ecl-lang-select-sites__link .ecl-lang-select-sites__label"
               page header: ".ecl-page-header__body"
+              page header title: ".ecl-page-header__title"
+              page header introduction: ".ecl-page-header__intro"
```
Comment (ademarco) [inline behat.yml.dist]: Do we need these two?

File: modules/europa_demo_core/src/Plugin/PageHeaderMetadata/LandingPagePageHeaderMetadata.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      'introduction' => $introduction,
+    ];
+
+    $cacheability = new CacheableMetadata();
+    $cacheability
+      ->addCacheableDependency($entity)
+      ->addCacheContexts(['route'])
+      ->applyTo($metadata);
+
+    return $metadata;
```
Comment (ademarco) [inline modules/europa_demo_core/src/Plugin/PageHeaderMetadata/LandingPagePageHeaderMetadata.php]: Maybe this way is more compact and we reuse the cache metadata set in the parent?

```
  /**
   * {@inheritdoc}
   */
  public function getMetadata(): array {
    $metadata = parent::getMetadata();

    $entity = $this->getEntityFromCurrentRoute();
    if (!$entity->get('field_introduction')->isEmpty()) {
      $metadata['introduction'] = $entity->get('field_introduction')->getString();
    }
    
    return $metadata;
  }
```

Comment (imanoleguskiza) [reply modules/europa_demo_core/src/Plugin/PageHeaderMetadata/LandingPagePageHeaderMetadata.php]: Looks good, I'll add it :)

File: behat.yml.dist (hunk 1 above)
Comment (imanoleguskiza) [reply behat.yml.dist]: Not anymore, I forgot to take them out

**Comment by ademarco** [comment]
Approved! Moving to +1.

**Pull request merged**